dask/array/reductions.py

```diff
@@ -23,9 +23,9 @@
     unknown_chunk_message,
 )
 from dask.array.creation import arange, diagonal
-
-# Keep empty_lookup here for backwards compatibility
```

Comment:
`empty_lookup` didn't seem to be used anywhere, and that has been the case for over a year.

Reply:
Am not aware of any usages of `empty_lookup`. Searching RAPIDS doesn't turn anything up either